buildSrc/src/main/kotlin/Android.kt

```diff
@@ -1,3 +1,9 @@
+/*
+   TODO: 04.12.2021 generally, it's a good idea to place all dependencies in such class.
+         Nevertheless, seems to me you made it too shattered, in the other words - overcomplicated.
+         You have a lot of classes with 3-5 rows. I've got the idea and totally understand why you moved
+         in this way, but it still to much, as for me.
+ */
```

Comment:
Yes, I already figured it out. It's convenient to keep related libraries in one file, but also we need one file for all other dependencies, which are single